gm4_soul_fragments/data/gm4_soul_fragments/functions/init.mcfunction

```diff
@@ -0,0 +1,15 @@
+#gm4_sf_died resets right after dying, while gm4_sf_rspwn resets after respawning
```

Comment:
Comments should be written with a ` ` after `#`, e.g.
`# comment text goes here`

Aditionally, headers should usually have the following format:
```
# @s = <human readable description of who' @s is in this function
# at <decription of where this command is run, e.g. @s or World Spawn>
# run from <caller>
```
Though, the second line may be omitted if the location of the file isn't of interest or variable.